src/cl/basic/cl_basic.c

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2020.  ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+#include "cl_basic.h"
+#include "cl/ucc_cl_log.h"
+#include "utils/ucc_malloc.h"
+
+static ucc_config_field_t ucc_cl_basic_lib_config_table[] = {
+    {"", "", NULL, ucc_offsetof(ucc_cl_basic_lib_config_t, super),
+     UCC_CONFIG_TYPE_TABLE(ucc_cl_lib_config_table)},
+
+    {NULL}
+};
+
+static ucs_config_field_t ucc_cl_basic_context_config_table[] = {
+    {"", "", NULL, ucc_offsetof(ucc_cl_basic_context_config_t, super),
+     UCC_CONFIG_TYPE_TABLE(ucc_cl_context_config_table)},
+
+    {"TEST_PARAM", "5", "For dbg test purpuse : don't commit",
```

Comment:
did you change your mind about committing? :)

Reply:
it is convenient for testing so far. i'll remove this as soon as we add 1st meaningful parameter. if there are objections i can remove it right now.